### PR TITLE
Now using datamodel v0.3.0; also upgraded to Java11

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

File: src/main/java/no/unit/nva/doi/transformer/MainHandler.java [hunk 1]
```diff
@@ -81,7 +87,8 @@ public void handleRequest(InputStream input, OutputStream output, Context contex
 
         try {
             UUID uuid = UUID.randomUUID();
-            Publication publication = converter.toPublication(dataciteResponse, uuid, owner);
+            URI publisherID = OrgNumberMapper.toCristinId(orgNumber);
+            Publication publication = converter.toPublication(dataciteResponse, uuid, owner, publisherID);
```
Comment (brinxmat) [inline src/main/java/no/unit/nva/doi/transformer/MainHandler.java:91]: is this still only a datacite response?

Comment (janbj) [reply src/main/java/no/unit/nva/doi/transformer/MainHandler.java:91]: No, not when Orestis is ready with his changes.

File: build.gradle [hunk 2]
```diff
@@ -13,18 +13,32 @@ repositories {
...
     implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.10.2'
     implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.10'
     implementation group: 'org.zalando', name: 'problem', version: '0.23.0'
     implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: '0.23.0'
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.5.2'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit5Version
     testImplementation group: 'org.mockito', name: 'mockito-core', version: '3.2.4'
     testImplementation group: 'com.github.stefanbirkner', name: 'system-rules', version: '1.19.0'
+    testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit5Version
+    testCompileOnly 'junit:junit:4.12'
```
Comment (axthosarouris) [inline build.gradle:34]: Do we really have tests that run only in vintage mode?

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,5 @@
 language: java
 jdk:
-  - openjdk8
```
Comment (axthosarouris) [inline .travis.yml:3]: remove from buildspec files as well

**Review by axthosarouris — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (janbj) [reply build.gradle:34]: I'll make a note about looking into it.

File: build.gradle [hunk 4]
```diff
@@ -13,18 +13,32 @@ repositories {
...
+    implementation group: 'com.github.BIBSYSDEV', name: 'nva-datamodel-java', version: 'v0.3.0'
     implementation group: 'com.amazonaws', name: 'aws-lambda-java-core', version: '1.2.0'
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.2'
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: '2.10.2'
     implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.10.2'
     implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.10'
     implementation group: 'org.zalando', name: 'problem', version: '0.23.0'
     implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: '0.23.0'
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.5.2'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit5Version
```
Comment (axthosarouris) [inline build.gradle:30]: why do we have this dependency twice? Also I think the correct dependency is ` testImplementation 'org.junit.jupiter:junit-jupiter:5.6.0' `

**Review by axthosarouris — APPROVED** [review]

**Pull request merged**